File: laboratorios/lab5/ex01.py
"""
Escreva um código que receba como entrada dois números (N,M) e, na saída, exiba na tela N eh multiplo de M se N for múltiplo de
M e, caso contrário, exiba N nao eh multiplo de M. O programa deve parar de repetir esse processo quando o valor de N digitado for igual -1.
"""

while True:
    n = int(input('Digite um valor: [-1] para sair '))
    m = int(input('Digite outro valor: '))

    if n == -1:
        break
    if m % n == 0:
        print(f'{m} é múltiplo de {n} :)\n')
    else:
        print(f'{m} não é múltiplo de {n}.\n')

File: lista02/ex01.c
/*
Faça um programa que receba como entrada um caractere e informe se ele é uma vogal ou não
*/

#include <stdio.h>
#include <ctype.h>

int main(){

    char caractere;

    printf("Digite um caractere: ");
    scanf("%c",&caractere);

    caractere = tolower(caractere);

    if (caractere == 'a' || caractere == 'e' || caractere == 'i' || caractere == 'o' || caractere == 'u'){
        printf("%c e uma vogal.",caractere);
    } else{
        printf("%c nao e uma vogal.",caractere);
    }

    return 0;
}